src/test/java/study/StringTest.java

```diff
@@ -16,4 +16,11 @@ void replace() {
         String[] str = "1,2".split(",");
         assertThat(str).contains("1","2");
     }
+
+    @Test
+    void substring_테스트() {
+        String str = "(1,2)";
+        String sub_str = str.substring(1, str.length()-1);
+        assertThat(sub_str).isEqualTo("1,2");
+    }
```

Comment:
```suggestion
    void substring_테스트() {
        String str = "(1,2)";
        String subStr = str.substring(1, str.length() - 1);
        assertThat(subStr).isEqualTo("1,2");
    }
```

space도 컨벤션이 있습니다. option + command + l 단축키를 사용하는 것도 좋은 방법일 것 같아요. 
그리고 자바는 카멜케이스를 사용합니다. 

https://velog.io/@leyuri/%ED%91%9C%EA%B8%B0%EB%B2%95-%EC%8A%A4%EB%84%A4%EC%9D%B4%ED%81%AC-%EC%BC%80%EC%9D%B4%EC%8A%A4-%ED%8C%8C%EC%8A%A4%EC%B9%BC-%EC%BC%80%EC%9D%B4%EC%8A%A4-%EC%B9%B4%EB%A9%9C-%EC%BC%80%EC%9D%B4%EC%8A%A4

Reply:
변수명은 축약어를 사용하지 않는 것이 좋아요. 이름은 길어져도 괜찮습니다.